webhook.go

```diff
@@ -7,14 +7,21 @@ import (
 
 // A Webhook represents an EasyPost webhook callback URL.
 type Webhook struct {
-	ID         string     `json:"id,omitempty"`
-	Object     string     `json:"object,omitempty"`
-	Mode       string     `json:"mode,omitempty"`
-	URL        string     `json:"url,omitempty"`
-	DisabledAt *time.Time `json:"disabled_at,omitempty"`
+	ID            string     `json:"id,omitempty"`
+	Object        string     `json:"object,omitempty"`
+	Mode          string     `json:"mode,omitempty"`
+	URL           string     `json:"url,omitempty"`
+	DisabledAt    *time.Time `json:"disabled_at,omitempty"`
+	WebhookSecret string     `json:"webhook_secret,omitempty"`
 }
 
-type createWebhookRequest struct {
+// UpdateWebhookOptions is used to specify parameters for updating EasyPost webhooks.
+type UpdateWebhookOptions struct {
+	URL           string `json:"url,omitempty"`
+	WebhookSecret string `json:"webhook_secret,omitempty"`
+}
+
+type createUpdateWebhookRequest struct {
```

Comment:
issue: Let's add a docstring to `createUpdateWebhookRequest` since we're here and it's missing and let's try to follow `CRUD` ordering (so swap the two order)

Reply:
Resolved